update!: Add LibreTranslate (#3032)
privacyguides/privacyguides.org: docs/language-tools.md

LanguageTool offers integration with a variety of [office suites](https://languagetool.org/services#text_editors) and [email clients](https://languagetool.org/services#mail_clients).

## Translation Tools

### LibreTranslate

<div class="admonition recommendation" markdown>

![LibreTranslate logo](assets/img/language-tools/libretranslate.png){ align=right }

> **LibreTranslate** is a free and open source machine translation web interface and API server. There are a number of [public instances](https://github.com/LibreTranslate/LibreTranslate?tab=readme-ov-file#mirrors), or you can host it yourself for complete privacy. It uses [Argos Translate](https://github.com/argosopentech/argos-translate) models on the backend for translations.
> 
> [:octicons-home-16: Homepage](https://libretranslate.com){ .md-button .md-button--primary }

Reviewer: ```suggestion
**LibreTranslate** is a free and open source machine translation web interface and API server. It uses [Argos Translate](https://github.com/argosopentech/argos-translate) models on the backend for translations.

You can access LibreTranslate through a number of public instances, with some that offer a [Tor](tor.md) onion service or an [I2P](alternative-networks.md#i2p-the-invisible-internet-project) eepsite. You can also host the software yourself for maximum control over the text submitted for translation.

[:octicons-home-16: Homepage](https://libretranslate.com){ .md-button .md-button--primary }
[:octicons-server-16:](https://github.com/LibreTranslate/LibreTranslate#mirrors){ .card-link title="Public Instances" }
```

Author: We would probably put additional paragraphs like this _below_ the card if we want to make this change.

> **LibreTranslate** is a free and open source machine translation web interface and API server. There are a number of [public instances](https://github.com/LibreTranslate/LibreTranslate?tab=readme-ov-file#mirrors), or you can host it yourself for complete privacy. It uses [Argos Translate](https://github.com/argosopentech/argos-translate) models on the backend for translations.
> 
> [:octicons-home-16: Homepage](https://libretranslate.com){ .md-button .md-button--primary }

Reviewer: ```suggestion
**LibreTranslate** is a free and open-source machine translation web interface and API server. It uses [Argos Translate](https://github.com/argosopentech/argos-translate) models on the backend for translations.

[:octicons-home-16: Homepage](https://libretranslate.com){ .md-button .md-button--primary }
[:octicons-server-16:](https://github.com/LibreTranslate/LibreTranslate#mirrors){ .card-link title="Public Instances" }
```

[:octicons-code-16:](https://github.com/LibreTranslate/LibreTranslate){ .card-link title="Source Code" }

</div>

> We use a self-hosted instance of LibreTranslate to automatically translate posts on our [forum](https://discuss.privacyguides.net) to multiple languages.

Reviewer: ```suggestion
You can use LibreTranslate through a number of public instances, with some that offer a [Tor](tor.md) onion service or an [I2P](alternative-networks.md#i2p-the-invisible-internet-project) eepsite. You can also host the software yourself for maximum control over the text submitted for translation.

We use a self-hosted instance of LibreTranslate to automatically translate posts on our [forum](https://discuss.privacyguides.net) to multiple languages.
```
